README.md

```diff
@@ -65,6 +65,21 @@ More examples are available [here](https://github.com/whylabs/langkit/tree/main/
 
 You can have more information about the different modules and their metrics [here](https://github.com/whylabs/langkit/blob/main/langkit/docs/modules.md).
 
+## Benchmarks
+
+| AWS Instance Type | Metric Module |      Throughput |
+|-------------------|:-------------:|----------------:|
+| c5.xlarge         | Light metrics | 2335 chats/sec  |
+|                   | LLM metrics   | 8.2 chats/sec   |
+|                   | All metrics   | 0.28 chats/sec  |
+
+|                   |               |                 |
+|-------------------|:-------------:|----------------:|
+| p3.2xlarge        | Light metrics | 1800 chats/sec  |
+|                   | LLM metrics   | 9.8 chats/sec   |
+|                   | All metrics   | 0.32 chats/sec  |
```

Comment:
Let's validate the GPU flavors of dependency closure for the p3. Might try with g4dn.xlarge instances since those latest supported types in the Databricks ML GPU runtimes

Reply:
Let's remove the GPU rows until we can investigate their behavior further, the c5.xlarge rows are still useful as a start